src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsOverview.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Box, Grid, useTheme } from '@mui/material';
+import { useAllChangeRequests } from '../../hooks/change-requests.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { isLeadership, isHead, ChangeRequest, Project, WorkPackage, wbsPipe } from 'shared';
+import PageBlock from '../../layouts/PageBlock';
+import { useAllProjects } from '../../hooks/projects.hooks';
+import { useAllWorkPackages } from '../../hooks/work-packages.hooks';
+import ChangeRequestDetailCard from '../../components/ChangeRequestDetailCard';
+import { useCurrentUser } from '../../hooks/users.hooks';
+
+const ChangeRequestsOverview: React.FC = () => {
+  const theme = useTheme();
+  const user = useCurrentUser();
+
+  const { isLoading, isError, error, data } = useAllChangeRequests();
```

Comment:
As you gave specific variable names in lines 22 and 23, can we have detailed variable names for this line too? (as crLoading, crData, ... for ex)